guacamole/src/main/webapp/translations/ja.json

```diff
@@ -0,0 +1,867 @@
+{
+    
+    "NAME" : "Japanese",
+    
+    "APP" : {
+
+        "NAME"    : "Apache Guacamole",
+        "VERSION" : "1.0.0",
+
+        "ACTION_ACKNOWLEDGE"        : "OK",
```

Comment:
If this is the Japanese translation for "OK" then this should stay, but values that are not being translated should be left out.